projects/addon-charts/components/line-chart/line-chart.component.ts

```diff
@@ -88,6 +88,10 @@ export class TuiLineChartComponent {
     @tuiDefaultProp()
     dots = false;
 
+    @Input()
+    @tuiDefaultProp()
+    dotContent: PolymorpheusContent = '';
```

Comment:
Maybe we can just change type of `dots` to `boolean | PolymorpheusContent`? Also make sure to add context type.